ipynb/examples/utils/testenv_example.ipynb

```diff
@@ -47,25 +131,32 @@
     "# Setup a target configuration\n",
     "conf = {\n",
     "\n",
-    "    # Platform and board to target\n",
+    "    # Platform\n",
     "    \"platform\"    : \"linux\",\n",
+    "    # Board\n",
     "    \"board\"       : \"juno\",\n",
     "\n",
+    "    # Target Android device ID\n",
```

Comment:
This is a linux (not android) platform so android device information is not necessary here. It just makes things unclear to have this here, even if it's commented.